src/main/java/frc/robot/Constants.java

```diff
@@ -211,7 +212,26 @@ public static final class EndgameConstants {
 
         public static final int smartCurrentLimit = 50;
 
-        public static final double encoderToMeters = Math.PI * Units.inchesToMeters(1.7567) / 20.0;
+        public static final TrapezoidProfile.Constraints climberProfileConstraints =
+                new TrapezoidProfile.Constraints(1, 2); // TODO: Find safe values for this
+
+        // TODO: Actual mass of robot
+        // 
+        public static final double simRobotMass = 53.0703; // 117 pounds to kg
+
+        // Constants for the climber PID controller
+        public static final double climberkP = 10.0; // TODO: Tune climber PID controller
+        public static final double climberkI = 0.0;
+        public static final double climberkD = 0.0;
+
+        // TODO: Tune feedforward values for climber
+        // Feedforward value for raising elevator
+        public static final double climberkFFClimber = 0.0;
+        // Feedforward value for lowering elevator, assuming we are now lifting entire
+        // weight of the robot
+        public static final double climberkFFRobot = 0.0;
```

Comment:
Obviously we should tune this on the robot, but just wanted to tell you my thought process in case you get to it when I'm not there.

Let's start the tune by finding kFFClimber, which will be the voltage at which the elevator just starts to move. Then put the climber into the up position and apply a downwards voltage to hook the chain. kFFRobot will be the minimum voltage needed to get the robot to just start to pull up on the chain and lift off the ground.

With those constants, you can start with the kP you have, which probably is okay but will likely have trouble getting us to the setpoint on either side without a little integral. I'd try with kI = 0 anyway at first just to confirm this. If indeed it slows down too much at the top/bottom, rather than boosting kP (which will add a ton of sensitivity and make things oscillate with sensor noise), I'd make kI = 1 and see if that helps make things snappier

Reply:
Another thing: in this PR can we add a test mode for finding those feedforwards (even if it's just a simple smart dashboard voltage entry thing that would be good)

Reply:
Other concerns are addressed, and I'll begin trying to add a test mode. Thanks for reviewing.